packages/rancher-vsphere/rancher-vsphere-csi/generated-changes/patch/Chart.yaml.patch

```diff
@@ -4,8 +4,8 @@
    catalog.cattle.io/namespace: kube-system
    catalog.cattle.io/os: linux,windows
    catalog.cattle.io/permits-os: linux,windows
--  catalog.cattle.io/rancher-version: '>= 2.6.0-0 <= 2.6.100-0'
+-  catalog.cattle.io/rancher-version: '>= 2.6.0-0'
 +  catalog.cattle.io/rancher-version: '>= 2.7.0-0 < 2.8.0-0'
    catalog.cattle.io/release-name: vsphere-csi
  apiVersion: v1
- appVersion: 2.5.1-rancher1
+ appVersion: 2.6.1-rancher1
```

Comment:
@brandond recently discovered the appVersion needs to be updated when updates are done to CSI/CPI charts. For CSI, this should be `2.6.2-rancher1.` Could we also update CPI in this PR as well?